db/migrations/postgres/000030_create_key_fields.up.sql

```diff
@@ -0,0 +1,15 @@
+BEGIN;
+
+ALTER TABLE batches ADD "key" VARCHAR(1024);
+UPDATE batches SET "key" = "";
```

Comment:
the empty string should be delimited by single quotes instead of double quotes, otherwise you'd get the following error during startup:

```
FF10163: Database migration failed: FF10163: Database migration failed: migration failed: zero-length delimited identifier at or near """" (column 28) in line 4: BEGIN;

ALTER TABLE batches ADD "key" VARCHAR(1024);
UPDATE batches SET "key" = "";
ALTER TABLE batches ALTER COLUMN "key" SET NOT NULL;

ALTER TABLE messages ADD "key" VARCHAR(1024);
UPDATE messages SET "key" = "";
ALTER TABLE messages ALTER COLUMN "key" SET NOT NULL;

ALTER TABLE tokenpool ADD "key" VARCHAR(1024);
UPDATE tokenpool SET "key" = "";
ALTER TABLE tokenpool ALTER COLUMN "key" SET NOT NULL;
```